source/server/parser.cpp

```diff
@@ -26,16 +27,22 @@ tokenList Server::parse(std::string buffer)
     for (std::vector<std::string>::iterator it = strList.begin(); it != strList.end(); it++)
     {
         line = *it;
+        if (line.at(0) == '/')
+        {
+            std::cout << "in [" << *line.begin() << "]" << std::endl;
+            line.erase(line.begin());
+            eof.erase(eof.begin());
```

Comment:
Se a linha tiver um '/' no início não esperamos um "\r\n"? Não era mais fácil procurar apenas o \n e se for preciso remover o \r?